docs/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.vectorstoreweaviate.md

```diff
@@ -0,0 +1,141 @@
+
+Here, the connection would be: AI agent (tools connector) -> Weaviate Vector Store node.
+
+### Use a retriever to fetch documents
+
+You can use the [Vector Store Retriever](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.retrievervectorstore.md) node with the Weaviate Vector Store node to fetch documents from the Weaviate Vector Store node. This is often used with the [Question and Answer Chain](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.chainretrievalqa/index.md) node to fetch documents from the vector store that match the given chat input.
+
+
+### Use the Vector Store Question Answer Tool to answer questions
+
+Another pattern uses the [Vector Store Question Answer Tool](/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolvectorstore.md) to summarize results and answer questions from the Weaviate Vector Store node. Rather than connecting the Weaviate Vector Store directly as a tool, this pattern uses a tool specifically designed to summarizes data in the vector store.
+
+	
+## Node parameters
+
+/// note | Multi Tenancy
+You can separate your data into isolated tenants for the same collection (eg different customers). For that, you must always provide a `Tenant` both for inserting and retrieving objects. [Read more about multi tenancy in Weaviate docs.](https://weaviate.io/developers/weaviate/manage-data/multi-tenancy)
+///
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-mode.md"
+
+--8<-- "_snippets/integrations/builtin/cluster-nodes/vector-store-rerank-results.md"
+
+
+<!-- vale from-write-good.Weasel = NO -->
+### Get Many parameters
+<!-- vale from-write-good.Weasel = YES -->
+
+* **Weaviate collection**: Enter the name of the Weaviate collection to use.
+* **Prompt**: Enter the search query.
+* **Limit**: Enter how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+
+This Operation Mode includes one **Node option**, the [Metadata Filter](#metadata-filter).
+
+### Insert Documents parameters
+
+* **Weaviate collection**: Enter the name of the Weaviate collection to use.
+
+### Retrieve Documents (As Vector Store for Chain/Tool) parameters
+
+* **Weaviate Collection**: Enter the name of the Weaviate collection to use.
+
+This Operation Mode includes one **Node option**, the [Metadata Filter](#metadata-filter).
+
+### Retrieve Documents (As Tool for AI Agent) parameters
+
+* **Name**: The name of the vector store.
+* **Description**: Explain to the LLM what this tool does. A good, specific description allows LLMs to produce expected results more often.
+* **Weaviate Collection**: Enter the name of the Weaviate collection to use.
+* **Limit**: Enter how many results to retrieve from the vector store. For example, set this to `10` to get the ten best results.
+
+This Operation Mode includes one **Node option**, the [Metadata Filter](#metadata-filter).
+
+## Node options
+
+### Metadata Filter
```

Comment:
This doesn't seem to match what I can see in the UI. There's an option called "Metadata Keys", but it seems like a much simpler interface than the one described here. This probably needs to be revised to match the current functionality unless I'm mistaken.